README.md

```diff
@@ -10,4 +10,4 @@ Describe how you approached to problem, and what tools and techniques you used t
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://happy-thoughts-api-estefania.herokuapp.com/
```

Comment:
The endpoints worked fine for me! Looking forward to seeing your API implemented in your frontend project! 🦄